### PR TITLE
possible solution for #74

### PR DESCRIPTION
**Pull request by mseele**
we also need an option to display the caret always

### CODE REVIEW

File: src/Multiselect.vue [hunk 1]
```diff
@@ -111,11 +111,11 @@
         </slot>
       </template>
 
-      <slot v-if="!hasSelected && caret && !busy" name="caret">
+      <slot v-if="(!replaceCaretWithClear || replaceCaretWithClear && !hasSelected) && caret && !busy" name="caret">
```
Comment (naftis) [inline src/Multiselect.vue:114]: @mseele should this variable be named `hideCaretOnSelect` ?

Comment (mseele) [reply src/Multiselect.vue:114]: Good idea. The problem is that the variable is responsible for displaying the clear action, too. I don't know if this is clear with the other name.
But I'm also mit very happy with my Name, btw.

Comment (naftis) [reply src/Multiselect.vue:114]: Yeah, I'm not sure what's the best name but was just wondering that the `.d.ts` prop name is different from this one

Comment (mseele) [reply src/Multiselect.vue:114]: You're write, I will change the .d.ts name tomorrow

Comment (mseele) [reply src/Multiselect.vue:114]: type definition name is fixed

**Comment by adamberecz** [comment]
@mseele in 2.0.0 caret will always be displayed when `caret: true`. I've added `canClear` which allows you to control the clear button instead.

**Pull request closed without merging**